wifiphisher/common/phishingpage.py

```diff
@@ -332,16 +332,16 @@ def search_template(self, name):
         dir_files = os.listdir(os.path.join(self._template_directory, name))
         # check all files in directory on...
         for files in dir_files:
-                    # html
+            # html
             if files.find(".html", 0) != -1:
                 html = True
             # config
             elif files.find("config.ini", 0) != -1:
                 config = True
```

Comment:
Do you think it is better if we can recursively find all the files under the template directory since this ensure that if in the future we want to change the file structure this function can still be used as well.

Reply:
Don't get it, sorry. If you want recursive search by name/extension then okay, but I'm not sure about create dir tree, hm.

Reply:
We don't need to create directory. What I mean is the following:

Currently this PR finds the `config.ini` and `.html` on the top-level of the directory like:
```
Template/config.ini
Template/index.html
```

But in the future if we change the Structure of the Template like the following:

```shell
Template/Configuration/config.ini
Template/HTML/index.html
```

Then you will fail to search the `config.ini` and `index.html`, right?

Reply:
Yes. I will try implement recursive search.